packages/pluggableWidgets/background-gradient-native/src/BackgroundGradient.tsx

```diff
@@ -33,11 +38,16 @@ export function BackgroundGradient({ name, colorList, angle, content, opacity, o
             style={({ pressed }) =>
                 StyleSheet.flatten([
                     styles.container,
-                    { opacity: onClick?.canExecute && pressed ? 0.5 : Number(opacity) }
+                    {
+                        opacity:
+                            onClick?.canExecute && pressed
+                                ? Number(styles.opacity ?? defaultStyle.opacity) * 0.3
+                                : Number(styles.opacity ?? defaultStyle.opacity)
```

Comment:
- I would suggest add a variable for `Number(styles.opacity ?? defaultStyle.opacity)` instead of repeating it.